splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -76,6 +76,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1933), new UpgradeScriptToUpdateViewForSYSCONGLOMERATEINSCHEMAS(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1938), new UpgradeScriptForTriggerWhenClause(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1941), new UpgradeScriptForTableColumnViewInSYSIBM(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1947), new UpgradeScriptBase(sdd,tc));
```

Comment:
We need to update the build number

Reply:
What should it be updated to?

Reply:
1948 if it's part of the next build, 1949 otherwise

Reply:
Done